src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/delete_interaction_choice_set_request.cc

```diff
@@ -31,6 +31,8 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <algorithm>
```

Comment:
@IGapchuk you do not need this library

Reply:
If we use std::uint32_t we need this library.